extensions/grpc/deployment/src/test/java/io/quarkus/grpc/client/MutipleStubsInjectionTest.java

```diff
@@ -32,20 +29,30 @@
                             MutinyGreeterGrpc.class, GreeterGrpc.class,
                             MutinyGreeterGrpc.MutinyGreeterStub.class,
                             HelloService.class, HelloRequest.class, HelloReply.class,
-                            HelloReplyOrBuilder.class, HelloRequestOrBuilder.class))
+                            HelloReplyOrBuilder.class, HelloRequestOrBuilder.class,
```

Comment:
FYI (and I found it recently), you can use `.addPackage(HelloService.class.getPackage())`, which avoid a long list of classes (not something to change)